### DISCUSSION TITLE
Constraints in UI Editor

### DISCUSSION THREAD

**Discussion by zakaria-chahboun** [supabase/supabase#13767] [Feature Requests]
Hi supabase developers!
it's better to let us add Constraints in the UI editor, especially the `ON DELETE/UPDATE CASCADE`.

Thanks :tulip:!

**Comment by kiwicopple** [comment]
Thanks for the feature request @zakaria-chahboun .

For this one we need to add it to the postgres api here: https://github.com/supabase/postgres-meta/issues/100. Any help is appreciated!
